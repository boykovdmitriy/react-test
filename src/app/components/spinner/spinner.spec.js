import React from 'react';
import {Spinner} from './index'

test('render spinner component', () => {
    const wrapper = shallow(
        <Spinner/>
    );
    expect(wrapper).toMatchSnapshot();
});