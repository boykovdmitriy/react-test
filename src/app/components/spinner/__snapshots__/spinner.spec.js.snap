// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render spinner component 1`] = `
<div
  className="loader"
/>
`;
